buildSrc/src/main/java/com/osacky/flank/gradle/FladlePluginDelegate.kt

```diff
@@ -40,6 +40,9 @@ class FladlePluginDelegate {
       // Must be done afterEvaluate otherwise extension values will not be set.
       project.dependencies.add(FLADLE_CONFIG, "${base.flankCoordinates.get()}:${base.flankVersion.get()}")
 
+      checkIfSanityAndValidateConfigs(base)
```

Comment:
let's move this to be the first line in `createTasksForConfig`
that way the next line can be removed as well.